ocs_ci/ocs/platform_nodes.py

```diff
@@ -32,7 +32,7 @@ class PlatformNodesFactory:
 
     """
     def __init__(self):
-        self.cls_map = {'AWS': AWSNodes, 'vsphere': VMWareNodes, 'aws': AWSNodes, 'baremetal': NodesBase}
+        self.cls_map = {'AWS': AWSNodes, 'azure': AZURENodes 'vsphere': VMWareNodes, 'aws': AWSNodes, 'baremetal': NodesBase}
```

Comment:
missing comma

Reply:
Also done't you need a class for AZURENodes?

Reply:
I added the NodesBase class for this.